File: templates/languages/python3/src/formula/formula.py
#!/usr/bin/python3
from colored import fg, attr


def Run(input1, input2, input3):
    print("Hello World!")
    print("%sYou receive {} in text.%s".format(input1) % (fg(2), attr(0)))
    print("%sYou receive {} in list.%s".format(input2) % (fg(1), attr(0)))
    print("%sYou receive {} in boolean.%s".format(input3) % (fg(3), attr(0)))

File: templates/languages/php/src/formula/formula.php
<?php
use Codedungeon\PHPCliColors\Color;

function Run($input1, $input2, $input3) {
    echo "Hello World! \n";
    echo Color::GREEN, "You receive $input1 in text. \n";
    echo Color::RED, "You receive $input2 in list. \n";
    echo Color::BLUE, "You receive $input3 in boolean. \n";
}
?>
